### DISCUSSION TITLE
[Bug]: Azure SSO Login via Open ID

### DISCUSSION THREAD

**Discussion by MarvinCraesEM** [danny-avila/LibreChat#3265] [Troubleshooting]
### What happened?

When logging in via Microsoft Entra ID, there seems so to be a bug with the librechat internal user assignment. If there is no first + last name specified the user will be randomly assigned to another user that has signed on with SSO. Got fixed when we specifically assigned first + lastname to the user.
Furthermore, we experienced that every user signed in via SSO share the same chats. Is that intended ? 

### Steps to Reproduce

1. Create a user in Azure without specifying first + last name
2. Login into Librechat via Open Id
3. Check user name bottom left

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is not expected and has never been experienced.

Did you follow this guide?

https://www.librechat.ai/docs/configuration/authentication/OAuth2-OIDC/azure

What does your host environment look like? Using docker, azure containers, azure cosmosdb, etc.?

**Reply by MarvinCraesEM** [reply]
First of all its fixed. 
Reproducing might be impossible, I guess we still had the old image without the "email can't be blank" fix. Which made it possible to login into the app via SSO without specifying the email in the azure profile. While debugging we upgraded the code and from that point on we couldn't login in via SSO due to the missing email in our profiles. Providing them fixed everything. Had to be some weird coincidences...
Sorry for the troubles and thanks for the fast response.
